gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/BasePage.java

```diff
@@ -47,13 +50,21 @@
 	/**
 	 * The current user
 	 */
+	@Getter
 	protected String currentUserUuid;
 
 	/**
 	 * The user's role in the site
 	 */
+	@Getter
 	protected GbRole role;
 
+	/**
+	 * The user's permissions in the site
+	 */
+	@Getter
+	protected List<PermissionDefinition> permissions;
```

Comment:
I think this might be better cached in the business service. I was going to do a sweep through the service and setup some caching for this sort of data so we can just hit the service and it sorts it out. Sticking stuff in the page as variables means it stays there forever as a copy of every page is kept.